woocommerce-paypal-payments.php

```diff
@@ -89,6 +89,7 @@ function() {
 			$proxy     = new ProxyContainer();
 			$provider  = new CompositeCachingServiceProvider( $providers );
 			$container = new CachingContainer( new DelegatingContainer( $provider ) );
+			$container = apply_filters( 'woocommerce_paypal_payments_built_container', $container );
 			$proxy->setInnerContainer( $container );
```

Comment:
I would suggest use a Action (`do_action`) with the Container after Line 93 and pass the proxy Container, because this Container is already readonly. I would only allow to change the Container if you provide a Module, then you can subscribe to the modules hook, where you can change the Container via a Module.

Reply:
Ok, updated.